lib/formatters/srt_formatter.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Subtitles.SrtFormatter do
+  def format(subtitles) do
+    (subtitles ++ [""])
```

Comment:
Why make this so convoluted? Just add another new-line pair at the end of formatting?

Reply:
Good point, went code blind here